manifests/init.pp

```diff
@@ -398,6 +393,13 @@
     }
   }
 
+  if $pulp or $pulp_master {
+    class { 'crane':
```

Comment:
Would it make sense to move this into `https://github.com/Katello/puppet-capsule/pull/40/files#diff-60ae41fd0a31977447947f59940ee9a4R259`